src/surfaces/actionBar/actionBar.jsx

```diff
@@ -36,28 +53,410 @@ const propTypes = {
             iconSearch: PropTypes.shape({}),
             iconTable: PropTypes.shape({}),
         })),
+        md: PropTypes.oneOf(GRID_SIZES),
         search: PropTypes.shape({}),
+        sm: PropTypes.oneOf(GRID_SIZES),
+        xl: PropTypes.oneOf(GRID_SIZES),
     })),
     id: PropTypes.string,
+    /**
+     * Defines the `justify-content` style property.
+     * It is applied for all screen sizes.
+     */
+    justifyContent: PropTypes.oneOf([
+        'center',
+        'flex-end',
+        'flex-start',
+        'space-around',
+        'space-between',
+        'space-evenly',
+    ]),
     moduleType: PropTypes.oneOf(['drawer', 'page']),
     style: PropTypes.shape({}), // eslint-disable-line react/forbid-prop-types
+    toggleSmSearchVisbileClassName: PropTypes.func,
```

Comment:
"Visible" is misspelled in this prop name.  Should be `toggleSmSearchVisibleClassName`.

Reply:
fixed.

Reply:
Looks good.  Resolving.